source/2019-06-16-ember-3-11-released.md

```diff
@@ -0,0 +1,200 @@
+
+- [Release Dashboard](http://emberjs.com/builds/)
+- [The Ember Release Cycle](http://emberjs.com/blog/2013/09/06/new-ember-release-process.html)
+- [The Ember Project](http://emberjs.com/blog/2015/06/16/ember-project-at-2-0.html)
+- [Ember LTS Releases](http://emberjs.com/blog/2016/02/25/announcing-embers-first-lts.html)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 3.11
+Ember.js 3.11 is an incremental, backwards compatible release of Ember with bugfixes, performance improvements, and minor deprecations. There is COUNT (#) new feature, COUNT (#) deprecations, and COUNT (#) bugfixes in this version.
+
+#### New Features (4)
+
+**Forwarding Element Modifiers with `...attributes` (1 of 4)**
+
+Angle bracket component invocation was introduced in Ember.js 3.7. Aside from the syntatic differences, the angle bracket invocation syntax enabled passing HTML attributes to components, which can then be applied the underlying HTML element(s) in the component's layout using the `...attributes` syntax. This is also known informally as the "splattributes" feature.
+
+This features clarifies how the "splattributes" feature interact with element modifiers.
+
+For more information please refer to [the RFC](https://github.com/emberjs/rfcs/blob/master/text/0435-modifier-splattributes.md).
+
+**The `{{fn}}` helper (2 of 4)**
+
+The `{{fn}}` helper provides a way to pass arguments to actions, particularly with native classes and the `@action` decorator.
+
+With the `@action` decorator, it is very straightforward to pass actions to other components ("data-down, actions-up"):
+
+```js
+import Component from "@ember/component";
+import { action } from '@ember/object';
+
+class CounterComponent extends Component {
+  count = 0;
+  
+  @action
+  countUp(increment = 1) {
+    this.incrementProperty("count", increment);
+  }
+}
+```
+ 
+```hbs
+Current count: {{this.count}}
+
+<MyButton @click={{this.countUp}}>Add One</MyButton>
+```
+
+The `@action` decorator ensures that the `this` value in the `countUp` action will always be the component instance, which allows it to be freely passed to other components. Where this falls short is when we need to pass arguments to these actions.
+
+The `{{fn}}` helper provides a way to pass arguments into actions and "bundle" them up, so that they can be passed around to other components and still retain the provided arguments when called:
+
+```hbs
+Current count: {{this.count}}
+
+{{!-- when not passing arguments, these are equivalent --}}
+<MyButton @click={{this.countUp}}>Add One</MyButton>
+<MyButton @click={{fn this.countUp}}>Add One</MyButton>
+
+{{!-- calls this.countUp(10) when clicked --}}
+<MyButton @click={{fn this.countUp 10}}>Add Ten</MyButton>
+```
+
+In addition to the basic use case shown here, the `{{fn}}` helper supports other advanced use cases, such as adding more arguments to an existing function.
```

Comment:
I asked my coworker to review, and she suggested adding a quick example here.

```suggestion
In addition to the basic use case shown here, the `{{fn}}` helper supports other advanced use cases, such as adding more arguments to an existing function.

 ```hbs
 {{#let (fn this.log "hello") as |hello|}}
   {{!-- calls this.log("hello", "world") --}}
   <MyButton @click={{fn hello "world"}} />
   
   {{!-- calls this.log("hello", "Tomster", "Zoey") --}}
   <MyButton @click={{fn hello "Tomster" "Zoey"}} />
 {{/let}}
 ```
```

(I had to put a space in front of the codeblock for the suggestion syntax to "work", but that probably means you can't just apply it without editing.)